gui/wxpython/mapwin/buffered.py

```diff
@@ -1182,7 +1182,7 @@ def MouseDraw(self, pdc=None, begin=None, end=None):
             r.Inflate(4, 4)
             try:
                 pdc.ClearId(boxid)
-            except:
+            except (KeyError, ValueError, AttributeError):
```

Comment:
I tested the ClearId function and it's not giving any error and I don't see how it could, so maybe removing the try except may be needed.